build

```diff
@@ -15,7 +15,7 @@ sleep 1.5
 rm -Rf output
 mkdir -p output/posts
 cp -r static output/static
-cp static/favicon.ico output/favicon.ico
+cp -r static/favicon output/favicon
```

Comment:
we must also have favicon.ico in the root directory, don't remove that

Reply:
I moved the favicon to `favicon` directory. Should be work. Right??

Reply:
@pirate did you check all the code? It should work fine.

Reply:
it will work, but the standard is to have favicon.ico in the root dir, and many browsers will still issue a request to favicon.ico even if the correct headers set it elesewhere, so it should always be present.